Modules/EMCAL/src/DigitCheck.cxx

```diff
@@ -0,0 +1,144 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   DigitCheck.cxx
+/// \author Cristina Terrevoli
+///
+
+#include "EMCAL/DigitCheck.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include <fairlogger/Logger.h>
+// ROOT
+#include <TH1.h>
+#include <TH2.h>
+#include <TPaveText.h>
+#include <TList.h>
+#include <iostream>
+
+using namespace std;
+
+namespace o2::quality_control_modules::emcal
+{
+
+void DigitCheck::configure(std::string) {}
+
+Quality DigitCheck::check(const MonitorObject* mo)
+{
+  Quality result = Quality::Good;
+
+  if (mo->getName() == "digitAmplitudeEMCAL") {
```

Comment:
For all histograms mentioned here we are planning more checks based on the shape of the distribution. Therefore we would prefer to merge this part so that the gaps can be filled in the near future.

Reply:
it makes sense, thank you for the comment.